setup.py

```diff
@@ -121,7 +121,8 @@
                         'astropy>=2.0.3',
                         'scipy',
                         'pandas>=0.12.0',
-                        'matplotlib>=1.3'],
+                        'matplotlib>=1.3',
+                        'pytest'],  # Upstream bug fix
```

Comment:
I don't think forcing users to install pytest is a great idea, maybe it could be added to `extras_require`?

Reply:
unfortunately as it's currently a hard dependency for the functioning of `coordinates` I am not really happy with people potentially not having it.

Reply:
All the other stuff in `extras_require` are hard dependencies for other modules though. (the background here is I want to use some of the coordinate stuff in HelioPy, but don't want to depend on SunPy if it means lots of other dependencies, e.g. AstroPy only requires numpy to install)